### DISCUSSION TITLE
Request for LD-2461 Component for ESPHOME

### DISCUSSION THREAD

**Discussion by naveenkkn** [TillFleisch/ESPHome-HLK-LD2450#16] [Ideas]
@TillFleisch 
Thanks for great contribution on the LD-2450 Component, and it works great and seamlessly. 

To be able to track more Targets within a region, can you recommend adding more than one LD2450 Modules in an Array with different frequency and Timings?

In such a case, can we track 3 Targets each 3 Target X 3 Modules i.e. upto 9 targets in the same 5 Meter scale region? 

Also Please extend your component for LD-2461 as well. 

Thanks in Advance!

**Comment by naveenkkn** [comment]
https://drive.google.com/drive/folders/14_KgZpL4Th2LTRuq_W0X_ePjHu_Z6lcS

**Comment by TillFleisch** [comment]
> To be able to track more Targets within a region, can you recommend adding more than one LD2450 Modules
> In such a case, can we track 3 Targets each 3 Target X 3 Modules i.e. upto 9 targets in the same 5 Meter scale region?

2 overlapping modules likely would result in issues. Either from a HW point of view (interference) or from the software side. To the best of my knowledge there is no way to tell the sensor to track a specific person/are. Therefore, there is no way to guarantee that different sensors track different persons. Furthermore, occlusion will be a major problem, with which the LD2450 seems to struggle at times. 

> in an Array with different frequency and Timings?

I don't understand what you mean by frequency and timing, can you elaborate? As far as my understanding goes, these are not configurable options on the LD2450.

> Also Please extend your component for LD-2461 as well.

Not anytime soon. I got the LD2450s because there were cheap and I wanted to play around with them. They are an acceptable solution for my use-case, therefore I am not incentivized to invest more money/time. If there is a need for this feature someone can fork this repository (and open a PR).

**Reply by naveenkkn** [reply]
Thanks for taking time to respond. 

We can run 2 LD2450 Radars side by side in 2 different Frequency settings without interfering each other, but The problem would be to uniquely identify different targets on each of them.
